cmd/kubeadm/app/apis/kubeadm/fuzzer/fuzzer_test.go

```diff
@@ -16,11 +16,6 @@ limitations under the License.
 
 package fuzzer
 
-// TODO: Fuzzing rouudtrip tests are currently disabled in the v1.12 cycle due to the
-// v1alpha2 -> v1alpha3 migration. As the ComponentConfigs were embedded in the structs
-// earlier now have moved out it's not possible to do a lossless roundtrip "the normal way"
-// When we support v1alpha3 and higher only, we can reenable this
-
```

Comment:
This probably needs to be moved to #69058

Reply:
You are right, but I missed that train, so I'm fixing it here